src/components/dataset/schemaCreation/DatasetSchemaInformationView.tsx

```diff
@@ -0,0 +1,355 @@
+          Cloud Platform*
+        </label>
+        <Controller
+          name="cloudPlatform"
+          control={control}
+          rules={{ required: true }}
+          render={({ field }) => (
+            <Select
+              id="dataset-cloudPlatform"
+              className={classes.formInput}
+              {...field}
+              onChange={(event: any, change: any) => {
+                const cloudPlatform = event.target.value;
+                setRegionOptions(
+                  _.get(cloudPlatforms, cloudPlatform) === cloudPlatforms.gcp
+                    ? GCP_REGIONS
+                    : AZURE_REGIONS,
+                );
+                setValue('region', '');
+                field.onChange(event, change);
+              }}
+              placeholder="Cloud platform"
+            >
+              {_.map(cloudPlatforms, (value: string, key: string) => (
+                <MenuItem value={key} key={key}>
+                  {value}
+                </MenuItem>
+              ))}
+            </Select>
+          )}
+        />
+      </Grid>
+
+      <Grid item xs={6} className={classes.formFieldContainer}>
+        <label
+          htmlFor="dataset-region"
+          className={clsx(classes.formLabel, { [classes.formLabelError]: errors.region })}
+        >
+          Region*
+        </label>
+        <Controller
+          name="region"
+          control={control}
+          rules={{ required: 'Region is required' }}
+          render={({ field }) => (
+            <Autocomplete
+              id="dataset-region"
+              options={regionOptions}
+              className={classes.formInput}
+              isOptionEqualToValue={(option, value) => value.name === option.name}
+              renderInput={(params: any) => (
+                <TextField
+                  {...params}
+                  error={!!errors.region}
+                  helperText={errors.region ? errors.region.message : ''}
+                  placeholder="cloud region"
+                />
+              )}
+              {...field}
+              onChange={(_event: any, change: any) => {
+                field.onChange(change);
+              }}
+            />
+          )}
+        />
+      </Grid>
+
+      <Grid item xs={12} className={classes.formFieldContainer}>
+        <label
+          htmlFor="dataset-stewards"
+          className={clsx(classes.formLabel, { [classes.formLabelError]: errors.stewards })}
+        >
+          Stewards
+        </label>
+        <div className={classes.formFieldDescription}>
+          A Steward, or Data Owner, is the person who created the dataset. While they are ultimately
+          liable for the data, they can assign the hands-on data management to another person by
+          assigning the Custodian role.
+        </div>
+        <Controller
+          name="stewards"
+          control={control}
+          rules={{
+            validate: {
```

Comment:
Do you think that these validation rules could be reusable in other forms? Did you think about pulling them out into a helper file so that other forms could make use of them? (Especially ones like this one that have some meat, but are going to be identical for many email validation checks)

Reply:
Ahm, yeah, i can pull it out into a helper file